releasenotes/notes/steppable-optimizers-9d9b48ba78bd58bb.yaml

```
@@ -0,0 +1,95 @@
+---
+features:
+  - |
+    Created an optimizer subclass :class:`~.SteppableOptimizer`, which allows performing classical
+    optimizations step-by-step using the :meth:`~.SteppableOptimizer.step` method. These
+    optimizers implement the "ask and tell" interface which (optionally) allows to manually compute
+    the required function or gradient evaluations and plug them back into the optimizer.
+    For more information about this interface see: `ask and tell interface
+    <https://optuna.readthedocs.io/en/stable/tutorial/20_recipes/009_ask_and_tell.html>`_.
+    A very simple use case when the user might want to do the optimization step by step is for
+    readout:
+
+    .. code-block:: python
+            import random
```

Maybe this is why sphinx fails?
```suggestion
    .. code-block:: python
    
            import random
```